mne/datasets/_fetch.py

```diff
@@ -213,7 +215,7 @@ def fetch_dataset(
                     "You must agree to the license to use this " "dataset"
                 )
     # downloader & processors
-    download_params = dict(progressbar=True)  # use tqdm
+    download_params = dict(progressbar=True if tqdm else False)  # use tqdm
```

Comment:
I don't think we need `if tqdm` anymore -- it's in our installation requirements, so it should be there. So instead I think we can do
```suggestion
    download_params = dict(progressbar=logger.level <= logging.INFO)
```
This should be enough for it to respect the current `verbose` level.

Reply:
Yes, and I have to remove pooch and tqdm soft dependency at the top of the file. I'll do it today.